packages/plugin-utils/src/processing/registerFillProcessingComponents.ts

```diff
@@ -21,62 +35,140 @@ export default function registerFillProcessingComponents(
   }
 ): {
   canvasMenuComponentId: string;
+  dockComponentId: string;
 
   translationsKeys: {
+    inspectorBarLabel: string;
+    navigationBarLabel: string;
+    canvasBarLabel: string;
     canvasMenuLabel: string;
+    dockLabel: string;
   };
 } {
   const { pluginId, locations } = options;
   const metadata = new FillProcessingMetadata(cesdk.engine, pluginId);
 
-  const canvasMenuComponentId = getCanvasMenuComponentId(pluginId);
   const canvasMenuLabel = getI18nCanvasMenuLabel(pluginId);
+  const canvasMenuComponentIds = getCanvasMenuComponentIds(pluginId);
+  const canvasMenuComponentId = canvasMenuComponentIds[0];
+
+  const dockLabel = getI18nDockLabel(pluginId);
+  const dockComponentIds = getDockComponentIds(pluginId);
+  const dockComponentId = dockComponentIds[0];
```

Comment:
This "looks" weird. Why would we get a list but only consider the first one?

Reply:
We use the first one for the internal ids. We register the component for both ids.